### PR TITLE
Make luks2john.py compatible with Python 3 too

### PR DESCRIPTION
**Pull request by kholia**
Let's use this PR instead of https://github.com/magnumripper/JohnTheRipper/pull/2732.

### CODE REVIEW

**Comment by kholia** [comment]
@kyrogon It would be great if you could test that this PR works for you too.

File: run/luks2john.py [hunk 1]
```diff
@@ -49,7 +50,7 @@
 
 def af_sectors(blocksize, blocknumbers):
     af_size = blocksize * blocknumbers
-    af_size = (af_size + 511) / 512
+    af_size = (af_size + 511) // 512
```
Comment (magnumripper) [inline run/luks2john.py]: Are you sure about this change? The `+ 511` will ensure rounding upwards. Won't the `//` make it round upwards once again?

Comment (kholia) [reply run/luks2john.py]:
``` Python
af_size = blocksize * blocknumbers                                           
af_size = (af_size + 511) // 512                                             
af_size *= 512 
```

I tested this change experimentally and it works fine.

Under Python 3, `(af_size + 511) / 512` produces a float quantity which doesn't produce correct results later on.

Comment (magnumripper) [reply run/luks2john.py]: Wouldn't just `af_size // 512` produce a correct rounded-up figure?

Comment (kholia) [reply run/luks2john.py]: The `//` operator doesn't do any rounding-up.

```
$ bpython
>>> 511 // 512 
0
>>> 512 // 512
1
>>> 513 // 512
1
```

Comment (magnumripper) [reply run/luks2john.py]: Oh, it's floor division, not ceiling 😆 .  I had it backwards, just ignore me!

Comment (kholia) [reply run/luks2john.py]: No worries :-)

**Pull request merged**